desktop/main.js

```diff
@@ -124,13 +113,14 @@ const electronUpdater = browserWindow => ({
 });
 
 const mainWindow = (() => {
-    const loadURL = ELECTRON_ENVIRONMENT.isDev()
+    const loadURL = __DEV__
```

Comment:
Deleted `ELECTRON_ENVIRONMENT` and replaced `isDev` / `isProd` usages with `__DEV__`